### PR TITLE
Implement sytles for press release content

### LINKED ISSUE #20: [Press Releases] Implement styles for press release content
- [x] styles for link lists (see https://www.eder-gmbh.de/newstermine/pressemeldungen/detail/kundenmagazin-der-eder-weihnachtspost-2022/)
- [x] styles for CTA buttons with different colors  - lets only support primary & secondary button styles (see https://www.eder-gmbh.de/newstermine/pressemeldungen/detail/agratec-news-als-e-paper-zum-durchblaettern/ or https://www.eder-gmbh.de/newstermine/pressemeldungen/detail/baumarkt-prospekt-des-eder-profi-baumarkt-in-tuntenhausen-bad-endorf-und-wasserburg/)
- [x] styles for center headlines (see https://www.eder-gmbh.de/newstermine/pressemeldungen/detail/eder-inspektionswochen-in-der-kleingeraetewerkstatt/)
- [x] bullet lists (see https://www.eder-gmbh.de/newstermine/pressemeldungen/detail/eder-feiert-steyr-busfahrt-mit-tollem-rahmenprogramm-zum-steyr-werk-in-st-valentin/)
- [x] "Weitere News" = more news link
- [x] #50 can be done with this one as well

### PR DESCRIPTION
**Pull request by mtilburgadobe**
* styles for linked lists was already done(validated)
* styles for buttons also completed already (validated)
* added styles for funky headlines in the content (example, for h6)
* bullet lists were already sytled (validated)
* Added the Weitere News link to each content area
* Will address the missing fonts in issue #50

Fix #20 

Test URLs:
- Before: https://main--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/karrieretag-bei-eder
- After: https://issue-20-styles--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/karrieretag-bei-eder

- Before: https://main--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/kundenmagazin-der-eder-weihnachtspost-2022
- After: https://issue-20-styles--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/kundenmagazin-der-eder-weihnachtspost-2022

- Before: https://main--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/eder-inspektionswochen-in-der-kleingeraetewerkstatt
- After: https://issue-20-styles--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/eder-inspektionswochen-in-der-kleingeraetewerkstatt

### CODE REVIEW

**Comment by mtilburgadobe** [comment]
I don't get this lint error locally, and furthermore, it's incorrect as far as I can see.  It's complaining that 'main .section' is duplicate, but it's not, it's only in there once.

**Comment by mhaack** [comment]
For https://issue-20-styles--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/kundenmagazin-der-eder-weihnachtspost-2022 the red h6 headline has the wrong font

**Comment by mtilburgadobe** [comment]
> For https://issue-20-styles--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/kundenmagazin-der-eder-weihnachtspost-2022 the red h6 headline has the wrong font

@mhaack , yes... this is the missing font that I will add in #50

File: scripts/scripts.js [hunk 1]
```diff
@@ -53,6 +53,29 @@ function buildAutoBlocks(main) {
...
+  const sidebar = main.getElementsByClassName('section sidebar')[0];
+  if (pressemeldungen && sidebar) {
+    const contentSection = body.children[1].children[1];
+
+    const fragment = document.createDocumentFragment();
+    const p = document.createElement('p');
+    const a = document.createElement('a');
+    a.textContent = 'Weitere News';
+    a.href = '/newstermine/pressemeldungen';
+    a.target = '_blank';
```
Comment (mhaack) [inline scripts/scripts.js:71]: I know they have this on the original page, but that feels wrong.

**Review by mhaack — APPROVED** [review]

**Pull request merged**